cassandra/data/benchflow.cql

```diff
@@ -57,7 +58,7 @@ CREATE TABLE environment_data (
   network_interfaces map<text, uuid>,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), container_id, host_id, environment_data_id)
+  PRIMARY KEY ((experiment_id, trial_id), host_id, container_id, environment_data_id)
```

Comment:
[This query](https://github.com/benchflow/analysers/blob/dev/analysers/commons/commons.py#L179) would be slower if you don't cluster also for `container_name`